test/fakeintake/server/parser.go

```diff
@@ -17,7 +18,15 @@ func NewPayloadParser() PayloadParser {
 	return parser
 }
 
-func (fi *PayloadParser) getLogPayLoadData(payload api.Payload) string {
+func (fi *PayloadParser) getJsonPayload(route string) ([]api.ParsedPayload, error) {
+	payload, ok := fi.payloadJsonStore[route]
+	if ok {
+		return payload, nil
+	}
+	return nil, errors.New("invalid route")
```

Comment:
It would be great to use `fmt.Errorf` to add the value of the route in the error message.